nginx.conf

```diff
@@ -30,7 +30,7 @@ http {
     listen 80;
     server_name login.telescope.cdot.systems;
     location / {
-      proxy_pass http://telescope_staging/login;
+      proxy_pass http://telescope_staging:3000/auth/login;
```

Comment:
We also have the same flow for `/auth/logout`, should we add that too?

Reply:
I'm not sure.
I think `/auth/logout` will be handled in:
```
server {
    listen 80 default_server;
    server_name _;
    return 307 https://$host$request_uri;
  }
```
`/auth/login` needs to be mention so we can tell nginx where to redirect all the requests sent to `login.telescope`, but (I might be wrong about this) I think `/auth/logout` will always be part of a `dev.telescope` request and it will go through the `307`, like when we requests `posts`, `feeds`, etc.

Reply:
Make sense.  Let's go with what you have and fix it later if it's wrong.  I bet you're right, though.